test/checks/shared/duplicate-id.js

```diff
@@ -49,4 +49,11 @@ describe('duplicate-id', function () {
 		assert.isTrue(checks['duplicate-id'].evaluate.call(checkContext, node));
 	});
 
+	it('should allow overwrote ids', function () {
+		fixture.innerHTML = '<form data-testelm="1" id="target"><input name="id"></form>';
```

Comment:
This might be overkill for test code...but as stewards of accessibility it would be appropriate to have a label in the fixture.